vars/runPipeline.groovy

```diff
@@ -234,6 +230,17 @@ def testDownstreamProject (name) {
                     '''
                     break;
 
+                case 'dlang-community/libdparse':
+                    sh 'cd test && DMD=$DC ./run_tests.sh'
+                    break;
+
+                case 'dlang-community/dcd':
+                    sh '''
+                    git submodule update --init --recursive
```

Comment:
The travis.sh script does sth. similar, sure just running the travis.yml script fails?